### PR TITLE
Data binning

### PR DESCRIPTION
**Pull request by maxnoe**
This is refactoring of the qla module and adding new functions to bin data in a new model called `analysis`.

The qla now uses the our peewee models, so no more raw sql queries in our code.
I also expanded the possibilities allowing to get and plot data for more than one night.

I added a new binning function, to create excess rate plots in a given binning.

### CODE REVIEW

File: fact/analysis.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
+import pandas as pd
+import numpy as np
+import datetime
+
+
+def li_ma_significance(N_on, N_off, alpha=0.2):
```
Comment (jebuss) [inline fact/analysis.py]: could you add a docstring?

**Review by jebuss — APPROVED** [review]
LGTM

File: fact/analysis.py (hunk 1 above)
Comment (maxnoe) [reply fact/analysis.py]: done

**Comment by dneise** [comment]
Maybe mentioning these new features in the README would be good, so people get to know about them.

**Pull request merged**